src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -89,20 +91,18 @@ public BitbucketSCM(
             @CheckForNull List<GitSCMExtension> extensions,
             String gitTool,
             String serverId) {
-
-        this.id = isBlank(id) ? UUID.randomUUID().toString() : id;
         if (isBlank(serverId)) {
-            throw new BitbucketSCMException("A server is required", "serverId");
+            LOGGER.severe("Error creating the Bitbucket SCM: the server ID must not be blank");
```

Comment:
I think `severe` is too.. severe. This is user input error, we shouldn't bother the logs with this unless debug logging is turned on.. I think